contrib/golang/filters/http/source/cgo.cc

```diff
@@ -47,6 +59,9 @@ CAPIStatus envoyGoFilterHttpSendLocalReply(void* r, int response_code, void* bod
        details](std::shared_ptr<Filter>& filter) -> CAPIStatus {
         UNREFERENCED_PARAMETER(headers);
         auto grpcStatus = static_cast<Grpc::Status::GrpcStatus>(grpc_status);
+
+        // Deep clone the GoString into C++, since the GoString may be freed after the function
```

Comment:
Yep, we already have tests for `sendLocalReply` which covered this code path. But, it's hard to reproduce the bad case, it could be randomly. I have added force GC after `sendLocalReply`, which tries its best to reproduce the bad case. It could be reproduced in fuzz testing.

Reply:
Nice. good for me.